app/statsd.js

```diff
@@ -3,7 +3,7 @@ const statsd = require('node-statsd');
 const statsdClient = new statsd({host: process.env.STATSD_HOST});
 
 statsdClient.getTimestamp = function() {
-  return Math.floor(new Date() / 1000);
+  return Math.floor(new Date());
```

Comment:
Nit, but you don't need `Math.floor()` since `new Date()` (or even better `Date.now()`) should return an integer and we're not doing division anymore.